#
# (c) Copyright 2015-2016 River Systems, all rights reserved.
#
# This is proprietary software, unauthorized distribution is not permitted.
#

uint32 TEMPERATURE=8
uint32 VOLTAGE=9
uint32 AVERAGE_CURRENT=11
uint32 INSTANTANEOUS_CURRENT=10
uint32 CHARGED_PERCENTAGE=13
uint32 AVERAGE_TIME_TO_EMPTY=18

uint8 id
float32 value